editor/editor_settings_dialog.cpp

```diff
@@ -772,6 +780,8 @@ EditorSettingsDialog::EditorSettingsDialog() {
 	shortcut_search_by_event->set_stretch_ratio(0.75);
 	shortcut_search_by_event->set_allowed_input_types(INPUT_KEY);
 	shortcut_search_by_event->connect("event_changed", callable_mp(this, &EditorSettingsDialog::_filter_shortcuts_by_event));
+	shortcut_search_by_event->connect("focus_entered", callable_mp(this, &EditorSettingsDialog::_on_listen_focus_changed));
```

Comment:
```suggestion
	shortcut_search_by_event->connect("focus_entered", callable_mp((AcceptDialog *)this, &AcceptDialog::set_close_on_escape).bind(false));
```
And same below but with `true` and remove the other method

Reply:
Makes sense.